src/lib/github.js

```diff
@@ -1,71 +1,28 @@
 import { checkStatus } from './request';
 
-function memoize(fn) {
-	const CACHE = {};
-	return key => CACHE[key] || (CACHE[key] = fn(key));
-}
+const baseUrl =
+	process.env.NODE_ENV === 'production'
+		? '/.netlify/functions/'
+		: 'http://localhost:9000/.netlify/functions/';
 
-// TODO: use GraphQL to avoid fetching so much data
-export const repoInfo = memoize(repo =>
-	fetch(`https://api.github.com/repos/${repo}`)
+export const repoInfo = repo =>
+	fetch(`${baseUrl}repo?repo=${repo}`, { mode: 'no-cors' })
```

Comment:
`mode:'no-cors'` prevents cors, I think we just want to avoid any cookies:

```suggestion
	fetch(`${baseUrl}repo?repo=${repo}`, { credentials: 'omit' })
```